docs/dev-env.md

```diff
@@ -166,10 +166,8 @@ $ docker run -it -v $PWD/demo:/var/fedimint -p 17240:17240 fedimintd:iqviraxy2cz
 write the files to the host file-system (e.g. config generation). `-p` is used to bind the host's port 17240 as the
 container's port 17240.
 
-To generate federation config and run the federation please see the integration tests and the `--help` output of
+To generate federation configs and run the federation please see the `run_dkg` and `start_federation` helper functions in [lib.sh](https://github.com/fedimint/fedimint/blob/master/scripts/lib.sh), as well as the `--help` output of
```

Comment:
Should we mention `source`ing here? By default we don't source lib in the dev env afaik (only in tmuxinator iirc).

Reply:
Added a sentence about sourcing `build.sh` and `lib.sh` here. This doesn't work for me, but if I hardcode the `SRC_DIR` parameter it does  (https://github.com/fedimint/fedimint/issues/1883).